biometricks/src/main/java/com/handstandsam/biometricks/BiometricException.kt

```diff
@@ -6,7 +6,7 @@ import androidx.biometric.BiometricPrompt
 /**
  * Error for when the biometric prompt fails for whatever reason.
  */
-class BiometricException(
+data class BiometricException(
```

Comment:
Curious what the purpose of this is? Exceptions don't normally implement `equals` in my experience.

Reply:
That's fine.  I thought you could do an `equals` but I guess that doesn't make sense for an Exception 👍